moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -760,10 +832,6 @@ bool MoveItConfigData::outputROSControllersYAML(const std::string& file_path)
   YAML::Emitter emitter;
   emitter << YAML::BeginMap;
 
-  // Start with the robot name  ---------------------------------------------------
```

Comment:
According to Gazebo docs it is better to have each Robot controllers under the namespace : `robot_name`, but  `moveit_controller_manager` looks for the controllers under `/controller_list`.
So in favor of the `moveit_controller_manager` the controllers and `/controller_list` will be with absolute namespace e.g. robot name will no longer be the name space of the `controller_list`.